lib/rubocop/cop/style/hash_slice.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for usages of `Hash#reject`, `Hash#select`, and `Hash#filter` methods
+      # that can be replaced with `Hash#slice` method.
+      #
+      # For safe detection, it is limited to commonly used string and symbol comparisons
+      # when using `==`.
+      # And do not check `Hash#delete_if` and `Hash#keep_if` to change receiver object.
```

Comment:
This part was copied from `HashExcept` docs, but I don't fully understand what this sentence means, it doesn't feel clear enough. Perhaps the docs should be updated?

Reply:
👍 I updated the wording in both cops, thanks.